src/static/js/modules/clear-form-buttons.js

```diff
@@ -0,0 +1,46 @@
+/* ==========================================================================
+   Clear form button
+   - Clear checkboxes and selects
+   - Clear Chosen.js elements
+   - Clear jquery.custom-input elements
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  $( '.js-form_clear' ).on( 'click', function() {
+    var $this = $( this ),
+        $form = $this.parents( 'form' );
+
+    // Clear text inputs
+    $form.find( 'input[type="text"]' ).val('');
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/bebb217d926085fc87d071fa3cb51ac548f9dc34